packages/clay-date-picker/src/__tests__/IncrementalInteractions.tsx

```diff
@@ -357,13 +357,9 @@ describe('IncrementalInteractions', () => {
 			);
 
 			const input: any = getByLabelText(ariaLabels.input);
-			const inputTimePicker: any = getByTestId('time-picker-input');
+			const hoursEl = getByTestId('hours') as HTMLInputElement;
 
-			fireEvent.change(inputTimePicker, {
-				target: {
-					value: '00:20',
-				},
-			});
+			fireEvent.keyDown(hoursEl, {key: '10'});
```

Comment:
nitpick, could you store the key value as a constant? Otherwise, I have no idea what key '10' is.

Reply:
Not sure about this, the key is just the number 10. 🙂

Reply:
does 10 specify a certain key? Like "enter" or "tab"?

Reply:
If not, we could also do something like `const ANY_KEY_CODE = '10'`

Reply:
Oh no, for that case I'm just wondering what number was pressed and not the key number. Maybe you are confusing it with `keyCode`.

Reply:
ahh yeah im mistaking it with keyCode